### DISCUSSION TITLE
关于新版本c++ pipeline的性能测试

### DISCUSSION THREAD

**Discussion by cyenxchen** [alibaba/ilogtail#1181] [Help] [answered]
看到新版本支持c++的处理插件了, 想了解下正则,json这俩处理插件的性能测试, 官方有相关测试结果不?

**Comment by messixukejia** [comment]
C++可以用后就可以对齐sls官方文档里的数据了https://help.aliyun.com/zh/sls/user-guide/configure-the-startup-parameters-of-logtail?spm=a2c4g.11186623.0.0.26d9320f67GSQ4

有兴趣的话，可以一起来测试下。

**Comment by yyuuttaaoo** [comment, marked as answer]
正则目前比Go模式快20%，JSON、多行切分等后面会再补充。
